README.md

```diff
@@ -16,7 +16,7 @@ the square brackets (for multiple-choice questions).
 What does it mean for an HTML tag to be "semantic"? Give an example of a short snippet of HTML written using semantic tags and non-semantic tags.
 
 ```text
-Your answer here.
+"Semantic" HTML means there's a function/purpose to the html rather than just the way it looks.  example: "<h1>" are semantic while "<strong>" are not. 
```

Comment:
✅ But actually, `<strong>` is considered to be semantic. (I don't entirely agree, but oh well.)